### ISSUE TITLE
No name was provided for external module '..srcextradashboard' in output.globals – guessing 

### ISSUE THREAD

**Issue by SirReaDy** [SoftwareBrothers/admin-bro#195]
Hi, I'm trying to implement the dashboard from the test app, but when even I log in it gives this error.
```
'..srcextradashboard' is imported by .adminbro\.entry.js, but could not be resolved – treating it as an external dependency
No name was provided for external module '..srcextradashboard' in output.globals – guessing 'Component1'
```

It cretes an .entry.js file with the following content:
```
AdminBro.UserComponents = {};
import Component1 from "..srcextradashboard";
AdminBro.UserComponents.Component1 = Component1;
```
When ever i change to the right path  upon launching it passes "srcextradashboard" instead if ../src/extra/dashboard

 "admin-bro": "^1.3.5",

**Comment by wojtek-krysiak** [comment]
hi - it is related to #136 - please try to use `admin-bro@1.4.0-beta.15` and let me know if that helped

**Comment by SirReaDy** [comment]
I was fixed with 1.4.0-beta.15, but I get new error and I'm kinda confused.
`{ SyntaxError: Unexpected token (42:6) in D:\admin\src\extra\dashboard.js
    at Object.pp$4.raise (D:\admin\node_modules\acorn\dist\acorn.js:2836:15)
    at Object.pp.unexpected D:\admin\node_modules\acorn\dist\acorn.js:689:10)
    at Object.pp$3.parseExprAtom (D:\admin\node_modules\acorn\dist\acorn.js:2260:12)
    at Object.parseExprAtom (D:\admin\node_modules\rollup\dist\rollup.js:11599:30)
    at Object.pp$3.parseExprSubscripts (D:\admin\node_modules\acorn\dist\acorn.js:2089:21)
    at Object.pp$3.parseMaybeUnary (D:\admin\node_modules\acorn\dist\acorn.js:2066:19)
    at Object.pp$3.parseExprOps D:\admin\node_modules\acorn\dist\acorn.js:2010:21)
    at Object.pp$3.parseMaybeConditional (D:\admin\node_modules\acorn\dist\acorn.js:1993:21)
    at Object.pp$3.parseMaybeAssign (D:\admin\node_modules\acorn\dist\acorn.js:1968:21)
    at Object.pp$3.parseParenAndDistinguishExpression (D:\admin\node_modules\acorn\dist\acorn.js:2332:30)
    at Object.pp$3.parseExprAtom (D:\admin\node_modules\acorn\dist\acorn.js:2220:43)
    at Object.parseExprAtom (D:\admin\node_modules\rollup\dist\rollup.js:11599:30)
    at Object.pp$3.parseExprSubscripts (D:\admin\node_modules\acorn\dist\acorn.js:2089:21)
    at Object.pp$3.parseMaybeUnary (D:\admin\node_modules\acorn\dist\acorn.js:2066:19)
    at Object.pp$3.parseExprOps (D:\admin\node_modules\acorn\dist\acorn.js:2010:21)
    at Object.pp$3.parseMaybeConditional (D:\admin\node_modules\acorn\dist\acorn.js:1993:21)
  pos: 919,
  loc:
   { file:
      'D:\\admin\\src\\extra\\dashboard.js',
     line: 42,
     column: 6 },
  raisedAt: 920,
  frame:
   '40: \r\n41:     return (\r\n42:       <React.Fragment>\r\n          ^\n43:         <DashboardHeader />\r\n44:         <WrapperBox style={{ marginTop: "-100px", zIndex: 2 }}>\r',
  id:
   'D:\\admin\\src\\extra\\dashboard.js',
  hook: 'transform',
  code: 'PLUGIN_ERROR',
  plugin: 'commonjs' }`

I'm getting this even with the default dashboard example.

"react": "^16.11.0",
"react-dom": "^16.11.0",
"@types/react": "^16.9.11",

Thank you for assisting me.

**Comment by wojtek-krysiak** [comment]
unexpected token in line 42 of your in D:\admin\src\extra\dashboard.js
